### PR TITLE
Final eval mtbench_branch

### PR DESCRIPTION
**Pull request by sallyom**
* This adds final eval mt_bench_branch 

* Also, adds a helpers package - for `launch_vllm`, `stop_vllm` -used in `mt_bench_branch`, `mmlu_branch`, and `mt_bench`, so need a central place for these.

* For mmlu_branch & mt_bench_branch, a taxonomy is required - this PR adds the taxonomy to the fake fixtures of SDG, to allow for mocking SDG with running final evaluation.

### CODE REVIEW

**Comment by sallyom** [comment]
[test run](https://rhods-dashboard-redhat-ods-applications.apps.ocp-beta-test.nerc.mghpcc.org/experiments/ilab/02978325-e127-4bef-9ebe-4f8f2e6c0875/runs/3d45ace0-bb95-43fc-9f2a-0ff42900ae3b)

**Comment by sallyom** [comment]
@MichaelClifford hold off on reviewing this for a bit - I'm importing defs from ilab rather than cp/pasting them here - will push that change after i test that it works

**Comment by sallyom** [comment]
@MichaelClifford this is ready for review, [test running now](https://rhods-dashboard-redhat-ods-applications.apps.ocp-beta-test.nerc.mghpcc.org/experiments/ilab/02978325-e127-4bef-9ebe-4f8f2e6c0875/runs/1d3b9739-04c1-4300-989a-a114f0c8b0c8)

**Review by MichaelClifford — APPROVED** [review]
LGTM! Thanks @sallyom

**Comment by MichaelClifford** [comment]
Will merge after the current run completes

**Review by MichaelClifford — APPROVED** [review]

File: sdg/faked/components.py
Comment (leseb) [inline sdg/faked/components.py:17]: Why did we need this change?

**Pull request merged**
